context/app/static/js/components/savedLists/SavedListScrollbox/style.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+import { PanelScrollBox } from 'js/shared-styles/panels';
+
+const SeparatedFlexRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: ${(props) => props.theme.spacing(0.5)}px;
+`;
+
+const FlexBottom = styled.div`
+  display: flex;
+  align-items: flex-end;
+`;
+
+const MaxHeightScrollbox = styled(PanelScrollBox)`
+  max-height: 400px;
```

Comment:
For the other scroll boxes, I think there's a shared styled component that gets precisely half a row, so it looks tidy, but it's very clear you're not seeing the whole thing.

Reply:
Increased the height to show more of the last panel. It's worth noting that this can vary.